Detectors/TRD/base/src/FeeParam.cxx

```diff
@@ -252,7 +252,7 @@ int FeeParam::getMCMfromPad(int irow, int icol) const
 
   if (irow < 0 || icol < 0 || irow > mgkNrowC1 || icol > mgkNcol)
     return -1;
-
+  //LOG(info) << "getMCMfromPad row : " << irow << " col : "<< icol << "answer : "<< (icol % (mgkNcol / 2)) / mgkNcolMcm + mgkNmcmRobInCol * (irow % mgkNmcmRobInRow)<< " with : " << (icol % (144 / 2)) << "/"<< 18 <<"+"<< 4 <<"*"<< (irow % mgkNmcmRobInRow);
```

Comment:
This should be removed

Reply:
removed and changed some others from info to debug as they should be.